apps/marketing/src/app/globals.css

```
@@ -1,5 +1,6 @@
 @import "tailwindcss";
 @import "@superset/ui/globals.css";
+@source "../../../../packages/ui/src/**/*.{ts,tsx}";
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)